internal/ingress/controller/template/template.go

```diff
@@ -1099,6 +1102,33 @@ func randomString() string {
 	return string(b)
 }
 
+func buildOpenTelemetry(c interface{}, s interface{}) string {
+	cfg, ok := c.(config.Configuration)
+	if !ok {
+		klog.Errorf("expected a 'config.Configuration' type but %T was returned", c)
+		return ""
+	}
+
+	servers, ok := s.([]*ingress.Server)
+	if !ok {
+		klog.Errorf("expected an '[]*ingress.Server' type but %T was returned", s)
+		return ""
+	}
+
+	if !shouldLoadOpenTelemetryModule(cfg, servers) {
+		return ""
+	}
+
+	buf := bytes.NewBufferString("")
+	buf.WriteString("opentelemetry_config /etc/nginx/opentelemetry.toml;\r\n")
+
+	if cfg.OpenTelemetryOperationName != "" {
+		buf.WriteString(fmt.Sprintf("opentelemetry_operation_name \"%s\";\n", cfg.OpenTelemetryOperationName))
```

Comment:
The first line uses `\r\n` but the second line uses `\n`. Should one be changed to match the other?

Reply:
Fixed, though the PR is still a draft and I'm trying to avoid a big bang PR. I am therefore opening several smaller ones, and this one probably shouldn't be reviewed so carefully (yet).

Reply:
> I am therefore opening several smaller ones, and this one probably shouldn't be reviewed so carefully (yet).

That's a great idea!